src/stream/src/executor/sink.rs

```diff
@@ -300,6 +302,7 @@ impl<F: LogStoreFactory> SinkExecutor<F> {
         input_data_types: Vec<DataType>,
         down_stream_pk: Vec<usize>,
         sink_chunk_buffer_size_metrics: LabelGuardedIntGauge<3>,
+        is_blackhole: bool,
```

Comment:
Can we make the name more specific, like `compact_chunk`? `is_blackhole` does not look intuitive to me.